### PR TITLE
Adding PyTao.

### PR DESCRIPTION
**Pull request by hhslepicka**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by hhslepicka** [comment]
@ChristopherMayes please confirm that you accept to be a maintainer of this feedstock to-be.

**Comment by ChristopherMayes** [comment]
@hhslepicka Yes I can be a maintainer.

**Comment by hhslepicka** [comment]
@conda-forge/help-python I am in doubt for the license if it should be `GPL-3.0-only` or `GPL-3.0-or-later`. Could you please advise?
The license file is here: https://github.com/bmad-sim/pytao/blob/master/LICENSE

**Comment by isuruf** [comment]
> PyTao is released under the GNU General Public License Version 3

`GPL-3.0-only`

**Comment by hhslepicka** [comment]
@isuruf thank you for your guidance. I updated the license via https://github.com/conda-forge/staged-recipes/pull/14648/commits/9125e008990fe6975591372e27c8f1d3943cc4f8. This one is good to go.

**Comment by hhslepicka** [comment]
@conda-forge/help-python this is good for a review/merge. Thank you

File: recipes/pytao/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - h5py
+    - numpy
+    - pexpect
```
Comment (isuruf) [inline recipes/pytao/meta.yaml:23]:
```suggestion
```

**Pull request merged**